llama.cpp

```diff
@@ -2308,6 +2312,9 @@ struct llama_model_loader {
             use_mmap = false;
         }
 
+#ifdef GGML_USE_KOMPUTE
+        use_mmap = false;
```

Comment:
Maybe add a warning about forcing `mmap = false`

Reply:
Why is this necessary?

Reply:
This seems to not be necessary, so I removed it.